src/tests/class_tests/openms/source/SimpleSVM_test.cpp

```diff
@@ -348,6 +348,11 @@ START_SECTION(regression_train_and_predict_on_separate)
 
   auto param = svm.getParameters();
   param.setValue("kernel", "RBF");
+  param.setValue("log2_C", ListUtils::create<double>("1,5"));
```

Comment:
I am still a bit puzzled that this takes so long... did we change anything regarding libSVM? do we always compile it ourself now etc.?

Reply:
ah ok I think it never showed because we ru our CI tests in parallel mode and it was finished when other tests were still running